### PR TITLE
Backend : API to GET List of All Warehouses

### PR DESCRIPTION
**Pull request by sabafarheen**

### CODE REVIEW

**Review by jodrasko — APPROVED** [review]
LGTM

File: server/routes/warehouses.js [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
+  const wareHouseList = fs.readFileSync("./data/warehouses.json");
+  return JSON.parse(wareHouseList);
+};
 
 // Fetch warehouse list end point
-wareHouseRouter.get('/', (_req, res) => {
-  
+wareHouseRouter.get("/", (_req, res) => {
+  let wareHouseList = readFile();
+  console.log(wareHouseList);
```
Comment (miriamnaki) [inline server/routes/warehouses.js]: The endpoint looks good. Consider deleting the console. logs when you are done.

File: server/routes/warehouses.js [hunk 2]
```diff
@@ -1,29 +1,41 @@
...
+const readFile = () => {
+  const wareHouseList = fs.readFileSync("./data/warehouses.json");
+  return JSON.parse(wareHouseList);
+};
 
 // Fetch warehouse list end point
-wareHouseRouter.get('/', (_req, res) => {
-  
+wareHouseRouter.get("/", (_req, res) => {
+  let wareHouseList = readFile();
```
Comment (miriamnaki) [inline server/routes/warehouses.js:15]: I think returning a response with the wareHouseList variable on line 15 should return a list of all warehouses. You do not need to map and return each field.

**Comment by miriamnaki** [comment]
Also check the branch to which you are merging! It should be develop not main

**Review by miriamnaki — APPROVED** [review]

File: server/index.js [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
 app.use(express.json());
 
 const PORT = process.env.PORT || 8080;
 
-app.use(cors({
-  origin: process.env.CLIENT_URL
-}));
+app.use(
+  cors({
+    origin: process.env.CLIENT_URL,
```
Comment (jodrasko) [inline server/index.js:16]: The code looks good. The only suggestion I would make is to erase the comma at the end of line 16.

**Review by jodrasko — APPROVED** [review]

**Pull request merged**